ada-project-docs/wave_01.md

```diff
@@ -25,27 +25,22 @@ It's crucial for all APIs to be able to handle errors. For every required endpoi
 
 If something goes wrong, your API should return:
 - an appropriate [HTTP status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)
-- a list of errors
+- For POST and PUT requests, responses with 4XX response codes should also return an array of error messages to explain what is wrong with the request.
 
 For this project, the list of errors should be formatted like this:
 
 ```json
-{
-  "errors": {
-    "available_inventory": [
-      "can't be blank",
-      "is not a number"
-    ]
-  }
-}
+// for a video endpoint
+[
+   "title must be provided and it must be a string",
+   "total_inventory must be provided and it must be a number"
+]
 
-// ...or...
+// ...or...for a rental endpoint
 
-{
-    "errors": [
-        "Not Found"
-    ]
-}
+[
+   "video_id must match a video in the video store database"
+]
 
```

Comment:
In Task List we had the students return a dictionary as the error message. Did we want to stay consistent with that? This could also not matter much at all as errors can be returned as either, if so, disregard this comment 😆  
```suggestion
{
        "details": "Invalid data"
}
```

Reply:
That's a good point Audrey. I think I'm pretty ok with the requirements being different for this project. But to be fair I didn't take the time to write tests to verify error messages so maybe we should be ahead and make the requirements more simple?

Reply:
Just synced with Jasmine and updated requirements here to be explicitly very flexible